db-tests/src/test/java/gov/nasa/jpl/aerie/database/PlanCollaborationTests.java

```diff
@@ -2165,6 +2131,22 @@ void commitMergeCleansUpSuccessfully() throws SQLException{
     }
   }
 
+  private static void assertActivityEquals(final Activity expected, final Activity actual) {
```

Comment:
Can this be moved up to where the other helper methods are?

Reply:
Done :+1: